src/foam/nanos/export/PDFGoogleSheetsExportDriver.js

```diff
@@ -31,44 +31,75 @@ foam.CLASS({
     {
       class: 'String',
       name: 'title'
```

Comment:
as most all of the code is the same for GoogleSheetsExportDriver.js and PDFGoogleSheetsExportDriver.js will create some kind of base class a little bit later